e2etests/features/eyfs_cms_signin_page_links.feature

```diff
@@ -1,50 +0,0 @@
-##  Based upon DfE Signin to Supply-Teachers
```

Comment:
This should not have been deleted as will be used when CMS stories are picked up.

Reply:
I disagree

If there is a way to change these to pending tests, then that would work, but I've had a look and can't see how

As I've said before, when the tests are implemented, they can come back

It is not good to keep commented out code in the repo